docs/wdc_webengine.md

```diff
@@ -0,0 +1,61 @@
+
+**Resolution:** WDCs that use an old version of `wdclib` need to upgrade to the latest version. If you are a developer of a Web Data Connector, refer to the migration guide for the steps to upgrade from version 1.x, see [Upgrading from WDC Version 1.x]({{ site.baseurl }}\docs\wdc_upgrade). If you are already using a 2.x library, you should be able to just upgrade to a newer version of the library (for example, `wdclib-2.3.latest.js`). For more information, see [WDC Versions]({{ site.baseurl }}\docs\wdc_library_versions).
+
+### JavaScript console errors, such as `not a valid WDC` or `tableau.XXX not a valid object`
+
+**Details:** WDCs under WebKit were initialized in a synchronous manner, which prevented their code from executing until all bootstrapping code was complete. Under WebEngine, it is not possible to make this guarantee. If your WDC attempts to use the `window.tableau` object before the `init` method is called,problems might occur due to the API not being fully initialized.
+
+**Resolution:** For many WDCs, simply upgrading to the latest `wdclib` will help. We have released patched versions of 2.1.x, 2.2.x, and 2.3.x to accommodate this. Make sure the WDC is using `wdclib-2.1.latest.js`, `wdclib-2.2.latest.js`, or `wdclib-2.3.latest.js`. It is important that WDC developers be aware that they still may not reliably use the `window.tableau` object until the WDC has been initialized. For example, it is illegal to use `window.tableau.log` before the WDC has been initialized. There is no resolution to this latter issue; WDCs should simply not do this.
+
+### Mixed-mode HTTP(S) errors
+
+**Details**: Under WebKit used in Tableau 2019.3 and earlier, mixed-mode HTTP and HTTPS calls were legal. That is, a WDC was able to make an AJAX request from an HTTPS host to a HTTP host. In Tableau 2019.4 and later, WebEngine reports errors when this occurs.
+
+**Resolution:** Always use HTTPS endpoints. Do not use mixed-mode.
+
+### Problem: Cookies are not preserved between instances of the WDC connection dialog.
+
+**Details:** Previously cookies may have been preserved under Qt WebKit WDC connection dialog instances. This was never intentionally supported and does not work with Qt WebEngine.
+
+**Resolution:** Don't rely on cookies. Use `tableau.connectionData` to pass data and expect users may need to re-authenticate if you don't preserve session data.
+
+### Problem: Can't find WDC logs
+
+**Details:** WDC emits all sorts of helpful logging information, in addition to log messages written through the tableau.log JavaScript API.
+
+**Resolution:** WDC logging information has moved. In Tableau Desktop, the WDC logging can be found in the regular log file. On Tableau Server, the WDC logging remains in `tabprotosrv` log.
+
+### Problem: help my WDC doesn't work and I can't do anything to fix it! Can I still use WebKit (the old browser engine)?
```

Comment:
this isn't something we want to disclose publicly. we can provide this information through tech support instead.

Reply:
removed

Reply:
I removed the logging info has moved item and also removed the `LegacyWdcMode`  registry/plist info.